tablecloth/render.cpp

```diff
@@ -342,17 +342,8 @@ namespace cloth::render {
       double view_y = (double) (output_box->height - view_box.height) / 2 + output_box->y;
       view.move(view_x, view_y);
 
-      if (has_standalone_surface(view) &&
```

Comment:
why?

Reply:
This is ported from rootston. `wlr_output_set_fullscreen_surface` no longer exists